dev-cpp/tomlplusplus/tomlplusplus-3.4.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Header-only library for parsing  and serializing TOML"
+HOMEPAGE="https://github.com/marzer/tomlplusplus"
+SRC_URI="https://github.com/marzer/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND=""
+DEPEND="${RDEPEND}"
+
+src_configure() {
+    cmake_src_configure
+}
```

Comment:
drop src_configure() ,  as cmake eclass inherited, it's already the default, 

you can check /var/tmp/portage/dev-cpp/tomlplusplus-3.4.0/temp/eclass-debug.log for this